behave/configuration.py

```diff
@@ -605,7 +605,8 @@ def __init__(self, command_args=None, load_config=True, verbose=None,
             load_configuration(self.defaults, verbose=verbose)
         parser = setup_parser()
         parser.set_defaults(**self.defaults)
-        args = parser.parse_args(command_args)
+        args, unknown_args = parser.parse_known_args(command_args)
+        self.unknown_args = unknown_args
```

Comment:
Can you not do `args, self.unknown_args = `?

Reply:
Good idea, I think I learned that just recently. Fixed it!